frontend/dockerui/context.go

```diff
@@ -113,6 +114,7 @@ func (bc *Client) initContext(ctx context.Context) (*buildContext, error) {
 		}
 		bctx.dockerfile = bctx.context
 	} else if (&gwcaps).Supports(gwpb.CapFrontendInputs) == nil {
+		bctx.localBuild = true
```

Comment:
This doesn't look correct condition for local build. The correct one seems to be in `MainContext` `if bctx.context != nil {` check